.github/pull_request_template.md

```diff
@@ -1,14 +1,93 @@
- - [ ] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
- - [ ] Ensure all `gulp` scripts successfully compile.
- - [ ] Update, remove, or extend all affected documentation.
- - [ ] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
+<!--- Feel free to remove any unused sections -->
 
 ## Obligatory GIF (super important!)
-![](path/to/gif)
+
+<!--- go to giphy.com, pick a gif, click share, copy gif link, paste within the () brackets below. -->
+
+![Obligatory GIF](path/to/gif)
+
+## :hammer_and_wrench: Type Of Change
+
+<!--- Tick or place an `x` for the type of change. Should match the type in the commit message / PR title
+in https://github.com/dialpad/dialtone/blob/staging/.github/COMMIT_CONVENTION.md -->
+
+These types will increment the version number on release:
+
+- [ ] Fix
+- [ ] Feature
+- [ ] Performance Improvement
+- [ ] Refactor
+
+These types will not increment the version number, but will still deploy to documentation site on release:
+
+- [ ] Documentation
+- [ ] Build system
+- [ ] CI
+- [ ] Style (code style changes, not css changes)
+- [ ] Test
+- [ ] Other (chore)
+
+## :book: Description
+
+<!--- Describe specifically what the changes are -->
+
+## :bulb: Context
+
+<!--- Describe the purpose of the changes -->
+<!--- Why did we make these changes? -->
+<!--- What problem(s) do they solve? -->
+
+## :pencil: Checklist
+
+<!--- Tick or place an `x` in all of the checkboxes that apply -->
+<!--- Remove checkboxes that do not apply -->
+
+For all PRs:
+
+- [ ] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
+- [ ] I have reviewed my changes.
+- [ ] I have added all relevant documentation.
+- [ ] I have considered the performance impact of my change.
+
+For all Vue changes:
+
+- [ ] I have added / updated unit tests.
+- [ ] I have made my changes in Vue 2 and Vue 3. Note: you may sync your changes from Vue 2 to Vue 3 (or vice versa) using the `./scripts/dialtone-vue-sync.sh` script.
+- [ ] I have validated components with a screen reader.
+- [ ] I have validated components keyboard navigation.
+
+For all CSS changes:
+
+- [ ] I have used design tokens whenever possible.
+- [ ] I have considered how this change will behave on different screen sizes.
+- [ ] I have visually validated my change in light and dark mode.
+- [ ] I have used gap or flexbox properties for layout instead of margin whenever possible.
+
+If new component:
+
+<!--- There are lots of things to remember when adding a new component to the system! This is so you don't forget any of them. -->
+
+- [ ] I am exporting any new components or constants from the index.js in the component directory.
+- [ ] I am exporting any new components or constants from the index.js in the root (either `packages/dialtone-vue2` or `packages/dialtone-vue3`).
```

Comment:
```suggestion
- I am exporting any new components or constants:
  - [ ] from the index.js in the component directory.
  - [ ] from the index.js in the root (either `packages/dialtone-vue2` or `packages/dialtone-vue3`).
```

Maybe we can rewrite this as a sub-item list to avoid repeating the same words, it's hard to read.

Reply:
looks good 👍